homeassistant/components/ovo_energy/strings.json

```diff
@@ -10,7 +10,8 @@
       "user": {
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
-          "password": "[%key:common::config_flow::data::password%]"
+          "password": "[%key:common::config_flow::data::password%]",
+          "account": "OVO account id (only add if you have multiple accounts)"
```

Comment:
Does the user know what the account id is? Is it not possible to let the user choose between existing accounts?